src/main/kotlin/io/github/anki/anki/controller/ControllerExceptionHandler.kt

```diff
@@ -1,24 +1,47 @@
 package io.github.anki.anki.controller
 
+import io.github.anki.anki.controller.exceptions.CardDoesNotExistException
+import io.github.anki.anki.controller.exceptions.DeckDoesNotExistException
+import io.github.anki.anki.controller.exceptions.DoesNotExist
 import org.slf4j.LoggerFactory
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
+import org.springframework.validation.FieldError
+import org.springframework.web.HttpRequestMethodNotSupportedException
+import org.springframework.web.bind.MethodArgumentNotValidException
 import org.springframework.web.bind.annotation.ExceptionHandler
-import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestControllerAdvice
+import java.util.LinkedHashMap
 
 @RestControllerAdvice
 class ControllerExceptionHandler {
+    @ExceptionHandler(HttpRequestMethodNotSupportedException::class)
```

Comment:
Not sure. HttpRequestMethodNotSupportedException is about user call server with method that server actually doesn't support.

Reply:
without this i had 500 when user called not existing path or method

Reply:
This is ok